lms/static/scripts/frontend_apps/components/SubmitGradeForm.tsx

```diff
@@ -19,25 +19,30 @@ import ValidationMessage from './ValidationMessage';
 
 export type SubmitGradeFormProps = {
   student: StudentInfo | null;
+
+  /**
+   * Scaling factor applied to grade values from the LMS to get the values
+   * entered by the user.
+   *
+   * LTI 1.1 only supports grade values between 0 and 1, which we then rescale
+   * to 0-{scoreMaximum} in the UI. LTI 1.3+ offer more flexibility (see [1]).
+   *
+   * [1] https://www.imsglobal.org/spec/lti-ags/v2p0
+   */
+  scoreMaximum: number | null;
```

Comment:
It isn't immediately obvious what `null` means here. It might be easier to use `scoreMaximum?: number` instead. I think that is more suggestive that a default value will be used in that case. This will also allow simplifying the type of the prop in `GradingControls` from `scoreMaximum?: number | null` to `scoreMaximum?: number`.

Reply:
Yep, makes sense.